DNN Platform/Library/Services/FileSystem/FileManager.cs

```diff
@@ -899,7 +899,7 @@ public virtual IFileInfo GetFile(int fileID, bool retrieveUnpublishedFiles)
             var file = DataCache.GetCache(strCacheKey);
             if (file == null)
             {
-                file = CBO.Instance.FillObject<FileInfo>(DataProvider.Instance().GetFileById(fileID, retrieveUnpublishedFiles));
```

Comment:
this and below changes same with this shouldn't change, otherwise will break all related unit tests.